packages/docs/src/styles/pages/layout.scss

```diff
@@ -27,6 +27,13 @@ body {
     :nth-child(2) {
       margin-top: 0;
     }
+    > h2 {
+      font-size: $font-size-3xl;
+      font-weight: normal;
+    }
+    > h3 {
+      font-size: $font-size-2xl;
+    }
```

Comment:
We now have some h4's. Do they need to be specified here?

Reply:
They probably should be - good callout!